mesheryctl/pkg/utils/error.go

```diff
@@ -235,35 +243,25 @@ func formatError(msg string, cmd cmdType) string {
 	}
 }
 
-func ErrAttachAuthToken(err error) error {
-	return errors.New(ErrAttachAuthTokenCode, errors.Alert, []string{err.Error()},
-		[]string{"Authentication token not found. Login with `mesheryctl system login` or supply a valid user token using the --token (or -t) flag."}, []string{}, []string{})
-}
-
 func ErrFailRequest(err error) error {
 	return errors.New(ErrFailRequestCode, errors.Alert, []string{"Failed to make a request"},
-		[]string{err.Error()}, []string{"Meshery server isn't running or not reachable"}, []string{"Please check your kubernetes cluster is running or not and Run `mesheryctl system restart`"})
+		[]string{err.Error()}, []string{"Meshery server isn't running or not reachable"}, []string{"Please check if your kubernetes cluster is running or your network connection and Run `mesheryctl system restart`"})
 }
 
 func ErrUnauthenticated() error {
 	return errors.New(ErrUnauthenticatedCode, errors.Alert, []string{"UnAuthenticated User"},
-		[]string{"You are UnAuthenticated to access any resource"}, []string{"You haven't login to meshery"}, []string{"Login with `mesheryctl system login`"})
+		[]string{"You are UnAuthenticated to access any resource"}, []string{"You haven't logged in to meshery"}, []string{"Please Login with `mesheryctl system login`"})
```

Comment:
It sounds polite, but there are two issues with "Please":

1) you didn't lowercase the following word ("Login").
2) don't use "Please", "thank you" or any colloquial or any derogatory or any complementary or any other style language than plain, direct, concise, factual, and systematic in nature.